### DISCUSSION TITLE
Windows 32 bit prebuilt realese?

### DISCUSSION THREAD

**Discussion by aurelVZ** [vlang/v#11667] [Ideas]
I am using windows 7 32 bit and i want to know why V simply not work on 32 bit windows.

1. is there any old version which is posible to compile on win 32bit os?

2. You must explain which C compiler you use for compiling  is that MingW ?

**Comment by vitalyster** [comment]
32-bit Windows is fully supported - https://github.com/vlang/v/wiki/Installing-a-C-compiler-on-Windows

**Comment by medvednikov** [comment]
Indeed it is, you just have to compile it yourself. It's as simple as running `make.bat` and waiting a couple of seconds.

There will be a 32 bit prebuilt release after 0.3.
